app/src/test/java/com/github/bjolidon/bootcamp/model/MagicSetTest.kt

```diff
@@ -0,0 +1,36 @@
+// ====================
+// MagicSetTest.kt
+// Tarjetakuna, 2023
+// ====================
+
+package com.github.bjolidon.bootcamp.model
+
+import org.junit.Test
+import org.junit.Assert.*
+
+class MagicSetTest {
+    @Test
```

Comment:
missing both args empty

Reply:
I don't think it's a good idea to test two different arguments in the same test. They are completely independent.

Reply:
I don't think it's a good idea to test two different arguments in the same test. They are completely independent.